docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -61,6 +80,14 @@ const config: SyntheticsConfig = {
 export default config;
 ----
 
+[discrete]
+[[synthetics-configuration-monitor]]
+== `monitor`
 
-NOTE: Synthetics tests configuration can only be used along with synthetics
-projects and is not available for inline suites.
+Default values to be applied to all monitors.
+
+`schedule` (`number`)::
+At which interval to run the test in seconds.
+
+`locations` (https://github.com/elastic/synthetics/blob/{synthetics_version}/src/dsl/monitor.ts#L36[`Array<SyntheticsLocationsType>`])::
```

Comment:
Are these the only `monitor` options that can be set in the project config file? I imagine things like `id` or `name` has to be done individually, but what about `screenshots`, `throttling`, and others?

Reply:
Thanks for clarifying @lucasfcosta! 

Based on the existing [config docs](https://www.elastic.co/guide/en/observability/current/synthetics-configuration.html#synthetics-config-file) and @vigneshshanmugam's [comment above](https://github.com/elastic/observability-docs/pull/1925#discussion_r897223744), should we exclude `params` and `playwrightOptions` from the global config docs for `monitors`?